### PR TITLE
Allow Participants to Reappraise

### PR DESCRIPTION
**Pull request by michaelwehrley**
Allow Participants to Reappraise

* Add reappraisal model
* Add migration to create `Reappraisal` model
* Add model specs
* Add feature specs to confirm submittion and errors
* Add index table to display past reappraisals
* Example partials to new form
* Update table html to include rows

[Finishes: #110760924]
[Finishes: #110760942]
[Finishes: #110761018]

### CODE REVIEW

File: db/migrate/20160329150220_create_reappraisals.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+class CreateReappraisals < ActiveRecord::Migration
+  def change
+    create_table :reappraisals do |t|
+      t.references :participant, foreign_key: true, index: true, null: false
+      t.string :description, limit: 10000, null: false
```
Comment (eschlange) [inline db/migrate/20160329150220_create_reappraisals.rb]: Is there a requirement for the specified limit or is this a performance optimization?

**Comment by eschlange** [comment]
Aside from my one question, everything looks good. Very similar to the other content providers created for Marigold. :shipit:

File: db/migrate/20160329150220_create_reappraisals.rb (hunk 1 above)
Comment (michaelwehrley) [inline db/migrate/20160329150220_create_reappraisals.rb]: @eschlange in the past we have started to limit the [size](https://github.com/cbitstech/think_feel_do_so/commit/959baf16fd19af5d9ba5e39c46007549a4f69347) and from the development site, it doesn't seem like they will need to input too large of a field.

Comment (michaelwehrley) [inline db/migrate/20160329150220_create_reappraisals.rb]: @ericcf What do you think I should do about adding limits?  Thanks!

Comment (ericcf) [inline db/migrate/20160329150220_create_reappraisals.rb]: My thought is that in general we should limit text columns until/unless told that more space is needed. It's less a storage issue and more a performance and display issue in my opinion.

**Comment by michaelwehrley** [comment]
@eschlange are you okay then if I merge?  Thanks!

**Pull request merged**